code/ecdsap256/Hacl.P256.fsti

```diff
@@ -332,7 +332,12 @@ val compression_not_compressed_form: b: lbuffer uint8 (size 64) -> result: notCo
     )
 
 
-[@ (Comment " Input: a point buffer (internal representation: uint8[64]), \n result: a point in not compressed form (uint8[33]).")]
+[@ (Comment "Convert from internal representation to 33-byte compressed form.
```

Comment:
Because the compressed form is rarely used it might be useful to add what the first byte is (0x02 or 0x03 depending Y)? Not too important though.

Reply:
There's some terminology discussion here: https://github.com/project-everest/hacl-star/blob/master/dist/gcc-compatible/Hacl_P256.h#L228 earlier in the file, so I'm assuming this has been introduced already?

Realistically we need to do a big overhaul of this file, and re-think documentation, APIs and naming conventions. In the meanwhile, I'm basically trying to fix up the most glaring omissions because I have some folks struggling with the API.

Let me know if you would like a reminder of what the compressed form is, in this specific function, or if the earlier explanation is ok.

Reply:
Ah, I missed the terminology. I guess it's fine for now.